### DISCUSSION TITLE
Get only the active country of the shop (ask to doc or more info)

### DISCUSSION THREAD

**Discussion by DidoMarchet** [saleor/saleor#10480] [General] [answered]
Hi guys,
I read about a new features that allows to display only the active countries of a shop.
Now when i get the countries, for the shipping address for example, it returns me the whole list.

I didn't save the article and then I'm asking you if you can give me more details or link to the doc.

Thanks in advance and kind regards,

Davide

**Comment by karolkielecki** [comment, marked as answer]
Hi @DidoMarchet, I'm not sure if I understood you correctly. Could you please tell me what you would like to achieve? 
Do you want to create a shipping zone with specific countries? https://docs.saleor.io/docs/3.x/dashboard/configuration/shipping#how-to

**Reply by DidoMarchet** [reply]
Hi @karolkielecki sorry for the ambiguity.

I found the request I told above https://github.com/saleor/saleor/pull/10111 
and I would like to know if is active and if there is a doc about.

We plan to update Saleor next week then I want to improve this feature.

Thanks in advance and kind regards,

Davide

**Reply by DidoMarchet** [reply]
Hi @karolkielecki ,
do you have any reference?

Really thanks in advance and kind regards,

Davide

**Reply by karolkielecki** [reply]
Hi @DidoMarchet, you can find the reference here: https://docs.saleor.io/docs/3.x/developer/api-reference/objects/channel#availableshippingmethodspercountry-shippingmethodspercountry.
